Userland/Services/WindowServer/WMClientConnection.cpp

```diff
@@ -24,6 +24,7 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+#include <LibGUI/Event.h>
```

Comment:
Is this change actually necessary?

Reply:
It isn't, that was an artifact of my old testing... sorry about that, I'll remove it now